adafruit_ds3231.py

```diff
@@ -107,3 +117,32 @@ def datetime(self, value):
         self.datetime_register = value
         self.disable_oscillator = False
         self.lost_power = False
+
+    @property
+    def temperature(self):
+        """Returns the last temperature measurement.  Temperature is updated
+        only every 64 seconds, or when a conversion is forced."""
+        return self._temperature / 4
+
+    def force_conversion(self):
+        """Forces a conversion and returns the new temperature"""
```

Comment:
`conversion` seems a bit vague. Is there a more detailed way of describing it? I expected it to be `force_temperature_conversion` but then calibration calls it.

Reply:
I'll rework this a bit and rename it.  (the checking for conversion-completed is wrong too)  It is forcing a temperature conversion, which is necessary for the new oscillator calibration to become effective.

Datasheet says:

> **Bit 5: Convert Temperature (CONV).**
> Setting this bit to 1 forces the temperature sensor to convert the temperature into digital code and execute the TCXO algorithm to update the capacitance array to the oscillator.
> This can only happen when a conversion is not already in progress. The user should check the status bit BSY before forcing the controller to start a new TCXO execution.
> A user-initiated temperature conversion does not affect the internal 64-second update cycle.
> A user-initiated temperature conversion does not affect the BSY bit for approximately 2ms.
> The CONV bit remains at a 1 from the time it is written until the conversion is finished, at which time both CONV and BSY go to 0. The CONV bit should be used when monitoring the status of a user-initiated conversion

and explains why triggering a conversion after setting the calibration value ("aging offset") is useful:

> The offset register is added to the capacitance array during a normal temperature conversion, if the temperature changes from the previous conversion, or during a manual user conversion (setting the CONV bit). To see the effects of the aging register on the 32kHz output frequency immediately, a manual conversion should be started after each aging register change.

`